src/validator.ts

```diff
@@ -0,0 +1,7 @@
+interface Validator {
```

Comment:
You will need to export this interface in the index.ts

Reply:
updated